### PR TITLE
TLP-51 Add job recommendations page

### PR DESCRIPTION
**Pull request by nepnep83**
Added data to the job recommendation page
Testing

### CODE REVIEW

File: app/templates/main/recommendation.html [hunk 1]
```diff
@@ -1,44 +1,33 @@
...
-
+    {{ govukTable(
+      {
+        'head': [{'text': 'Job title'}, {'text': 'Description'}, {'text': 'Link'}],
+        'rows': [
+          [{'text': job1}, {'text': desc1}, {'href': link1, 'text': 'Job 1'}],
+          [{'text': job2}, {'text': desc2}, {'href': link2, 'text': 'Job 2'}]
+        ]
+      }
+    )}}
```
Comment (Aidan23Smith) [inline app/templates/main/recommendation.html:30]: Change this, you forgot

File: requirements.txt [hunk 2]
```diff
@@ -2,14 +2,18 @@
 # This file is autogenerated by pip-compile with Python 3.10
 # by the following command:
 #
-#    pip-compile requirements.in
+#    pip-compile
```
Comment (Aidan23Smith) [inline requirements.txt:5]: Oh, you should probably run `pip-compile requirements.in` to use it properly

File: Backend/job_vacancies.py [hunk 3]
```diff
@@ -13,21 +13,23 @@ def get_claimant_info():
...
-    jobs = recommend_jobs.run(_range, interests_range, job_range)
+def get_recommend_jobs(jobs, interest):
+    jobs = recommend_jobs.run(jobs, interest, _range, interests_range, job_range)
     return jobs
 
 
 def find_vacancies(travel_distance, postcode, job):
+    print(type(travel_distance))
+    print(type(postcode))
+    print(type(job[0]))
```
[the remark this answers is not in the document]
Comment (nepnep83) [reply Backend/job_vacancies.py:24]: You are correct.

File: Backend/job_vacancies.py [hunk 4]
```diff
@@ -36,6 +38,9 @@ def run(job, _range):
         print(local_jobs[i]['link'], '\n')
     return local_jobs
```
[the remark this answers is not in the document]
Comment (nepnep83) [reply Backend/job_vacancies.py:40]: No, it is more readable with the blank line when looking at the result in terminal and doesn't affect what the user will see when on the website

File: Backend/job_vacancies.py [hunk 5]
```diff
@@ -36,6 +38,9 @@ def run(job, _range):
         print(local_jobs[i]['link'], '\n')
```
[the remark this answers is not in the document]
Comment (nepnep83) [reply Backend/job_vacancies.py:38]: I believe that is the same line of code as the previous comment

File: app/templates/main/recommendation.html (hunk 1 above)
Comment (nepnep83) [reply app/templates/main/recommendation.html:30]: So I did, thank you for your brilliant observation skills 😂.

File: requirements.txt (hunk 2 above)
Comment (nepnep83) [reply requirements.txt:5]: Done, thank you.

**Review by Aidan23Smith — APPROVED** [review]

**Pull request merged**